tests/unit/matrix/test_cubemeasure.py

```diff
@@ -1798,35 +1825,6 @@ def it_knows_its_weighted_counts(self):
 
         assert matrix.weighted_counts.tolist() == [[3, 2, 1], [6, 5, 4]]
 
-    def it_knows_its_zscores(self):
-        weighted_counts = np.array([[3, 2, 1], [6, 5, 4]])
-        matrix = _CatXCatMatrix(None, weighted_counts, None)
-
-        np.testing.assert_almost_equal(
-            matrix.zscores,
-            np.array([[0.41833001, 0.0, -0.48605555], [-0.41833001, 0.0, 0.48605555]]),
-        )
-
-    @pytest.mark.parametrize(
-        "weighted_counts, expected_value",
-        (
-            (
-                np.array([[1, 2], [0, 0]]),
-                np.array([[np.nan, np.nan], [np.nan, np.nan]]),
-            ),
-            (
-                np.array([[0, 2], [0, 4]]),
-                np.array([[np.nan, np.nan], [np.nan, np.nan]]),
-            ),
-        ),
-    )
-    def but_its_zscores_are_NaNs_for_a_deficient_matrix(
```

Comment:
This is a better test title than `it_does_not_calculate_zscores_when_matrix_is_defective`. It does in fact return a value and this title spells out what that is and why.